py/server/deephaven/parquet.py

```diff
@@ -382,23 +410,19 @@ def batch_write(
     max_dictionary_keys: Optional[int] = None,
     max_dictionary_size: Optional[int] = None,
     target_page_size: Optional[int] = None,
-    grouping_cols: Optional[List[str]] = None,
     generate_metadata_files: Optional[bool] = None,
+    index_columns: Optional[List[List[str]]] = None
 ):
     """ Writes tables to disk in parquet format to a supplied set of paths.
 
-    If you specify grouping columns, there must already be grouping information for those columns in the sources.
-    This can be accomplished with .groupBy(<grouping columns>).ungroup() or .sort(<grouping column>).
-
     Note that either all the tables are written out successfully or none is.
 
     Args:
         tables (List[Table]): the source tables
-        paths (List[str]): the destinations paths. Any non existing directories in the paths provided are
+        paths (List[str]): the destination paths. Any non-existing directories in the paths provided are
             created. If there is an error, any intermediate directories previously created are removed; note this makes
             this method unsafe for concurrent use
-        col_definitions (Optional[List[Column]]): the column definitions to use for writing, instead of the definitions
-            implied by the table. Default is None, which means use the column definitions implied by the table
+        col_definitions (List[Column]): the column definitions to use for writing.
```

Comment:
Was this a good change?

Reply:
So I made a mistake by changing it from List[Column] to Optional[List[Column]] earlier. But have reverted it back now.